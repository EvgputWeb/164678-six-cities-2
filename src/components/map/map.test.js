import React from 'react';
import Enzyme, {mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Map from './map.js';
import OFFERS from '../../mocks/offers.js';

Enzyme.configure({adapter: new Adapter()});


it(`Map render test`, () => {
  const div = global.document.createElement(`div`);
  global.document.body.appendChild(div);

  const citiesMap = mount(<Map list={OFFERS} />, {attachTo: div});

  expect(citiesMap).toMatchSnapshot();
});
