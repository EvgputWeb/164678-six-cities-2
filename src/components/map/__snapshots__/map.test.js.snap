// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders correctly 1`] = `
<div
  className="cities__right-section"
>
  <section
    className="cities__map map"
  />
</div>
`;
